Packs/SymantecDLP/Integrations/SymantecDLP/SymantecDLP.py

```diff
@@ -692,9 +698,12 @@ def fetch_incidents(client: Client, fetch_time: str, fetch_limit: int, last_run:
                 incident['attachment'] = attachments
 
             incidents.append(incident)
-            if incident_id == incidents_ids[-1]:
-                last_incident_time = incident_creation_time
-                last_incident_id = incident_id
+
+            last_incident_time = incident_creation_time
```

Comment:
It could be the incidents are not sorted. So I would save the new time only if it is larger that the last one saved.